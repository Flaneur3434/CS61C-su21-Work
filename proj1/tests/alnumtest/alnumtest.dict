     h3Ll0 hello
   G00dbye h#
             again bye
   5 %@&*)(
 2345                                                                                                       5432
                            3s                                                    error
3s4 alpha
