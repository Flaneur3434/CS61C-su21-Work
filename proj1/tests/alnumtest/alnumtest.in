#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
#3s4!!!   ** * * h3LL0 G00DBye AGAIN 2345! !2345 234 5
