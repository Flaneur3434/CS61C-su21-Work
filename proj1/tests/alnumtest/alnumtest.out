#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
#alpha!!!   ** * * h3LL0 h# bye 5432! !5432 234 %@&*)(
